example/index.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+import { printSchema, buildClientSchema, getIntrospectionQuery } from 'graphql';
+import TykGraphiQL from '@tyk-technologies/graphiql';
+import './common/styles/index.scss';
+
+const Example = () => {
+  const [selectedApi, setSelectedApi] = useState({
+    name: 'Countries',
+    url: 'https://countries.trevorblades.com/',
+  });
+
+  const [query, setQuery] = useState(`{
+    countries {
+      name
+    }
+  }
+  `);
+
+  const defaultSchema =
```

Comment:
I wouldn't enforce a default schema, this can be custome per project

Reply:
this is example @lghiur